### PR TITLE
update for v3

### PR DESCRIPTION
**Pull request by alexcos20**

### CODE REVIEW

File: 7/v0.3/README.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
+# OEP-7: Decentralized Identifiers
+
+```
+shortname:      7/DID
+name:           Decentralized Identifiers
+type:           Standard
+status:         Draft
+version:        0.3
+editor:         Alex Coseru <alex@oceanprotocol.com>
+contributors:   Aitor Argomaniz <aitor@oceanprotocol.com>
```
Comment (0x3bfc) [inline 7/v0.3/README.md:10]: Contributors section need to be updated.

File: 7/v0.3/README.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+* Define how Ocean assets, agents and domains can be modeled with a DID/DDO data model
+* Understand how DID hubs are formed, and how they integrate a business and storage layer
+
+## Specification
+
+Requirements are:
+
+* The DID resolving capabilities MUST be exposed in the client libraries, enabling to resolve a DDO directly in a totally transparent way
+* ASSETS are DATA objects describing RESOURCES under control of a PUBLISHER
+* KEEPER stores on-chain only the essential information about ASSETS
```
Comment (0x3bfc) [inline 7/v0.3/README.md:66]: is it KEEPER or just `OCEAN-CONTRACTS` / `DATATOKEN`

File: 7/v0.3/README.md [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+* PROVIDERS store the ASSET metadata off-chain
+* KEEPER doesn't store any ASSET metadata
```
Comment (0x3bfc) [inline 7/v0.3/README.md:68]: the same here

File: 7/v0.3/README.md [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+* access - describing how the asset can be downloaded
+* compute - describing how the asset can be computed upon
+
+Each service is distinguished by the `DDO.service.type` attribute.
+
+Each service has an `attributes` section where all the information related to the service is added. As mandatory content, the attributes section will have a `main` sub-section. This one is important because it must include all the mandatory information that a service has to provide.
+
+A part of the `attributes.main` sub-section, other optional sub-sections can be added (like: `attributes.curation` or `attributes.extra`) depending on the service type.
+
+Each service has an `dtCost` and `timeout` (in seconds) section describing the cost and how long the sevice can be used after payment. A timeout of 0 represents no time limit.
```
Comment (0x3bfc) [inline 7/v0.3/README.md:138]: Are we storing the timeout on-chain ? or will just keep it off-chain?

File: 7/v0.3/ddo-example.json [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+        }
+      },
+      {
+        "type": "access",
+        "index": 1,
+        "serviceEndpoint": "http://localhost:8030/api/v1/brizo/services/consume",
+        "templateId": "",
+        "attributes": {
+          "main": {
+            "name": "dataAssetAccessServiceAgreement",
```
Comment (0x3bfc) [inline 7/v0.3/ddo-example.json:119]: `dataAssetAccessService` not `dataAssetAccessServiceAgreement`

File: 7/v0.3/ddo-example.json [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+            "schema": "Binary Votting",
+            "isListed": true
+          }
+        }
+      },
+      {
+        "type": "access",
+        "index": 1,
+        "serviceEndpoint": "http://localhost:8030/api/v1/brizo/services/consume",
+        "templateId": "",
```
Comment (0x3bfc) [inline 7/v0.3/ddo-example.json:116]: what is the `templateId` ?

File: 7/v0.3/ddo-example.json [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+          "additionalInformation": {
+            "description": ""
+          }
+        }
+      },
+      {
+        "type": "compute",
+        "index": 2,
+        "serviceEndpoint": "http://localhost:8030/api/v1/brizo/services/compute",
+        "templateId": "",
```
Comment (0x3bfc) [inline 7/v0.3/ddo-example.json:134]: the same here for the `templateId`

File: 8/v0.5/ddo-example-algorithm.json [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+      }
+    },
+    {
+      "type": "access",
+      "index": 1,
+      "serviceEndpoint": "http://localhost:8030/api/v1/brizo/services/consume",
+      "templateId": "",
+      "attributes": {
+        "main": {
+          "name": "dataAssetAccessServiceAgreement",
```
Comment (0x3bfc) [inline 8/v0.5/ddo-example-algorithm.json:111]: the same here for the `Agreement`.

File: 7/v0.3/README.md (hunk 4 above)
Comment (alexcos20) [reply 7/v0.3/README.md:129]: no, we will keep it off-chain, only in the DDO

File: 7/v0.3/README.md (hunk 3 above)
Comment (alexcos20) [reply 7/v0.3/README.md]: removed all on-chain references, because now everything is off-chain

**Pull request merged**